<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationExportDialog</class>
 <widget class="QDialog" name="SimulationExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>仿真结果导出</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin" stdset="0">
    <number>9</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane {
    border: 1px solid #c0c0c0;
    top: 1px;
}
QTabBar::tab {
    padding: 4px 8px;
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="globalTab">
      <attribute name="title">
       <string>全局信息</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin" stdset="0">
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="globalGroupBox">
         <property name="title">
          <string>全局指标</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin" stdset="0">
           <number>9</number>
          </property>
          <item>
           <widget class="QTableWidget" name="globalTable">
            <property name="styleSheet">
             <string notr="true">QTableWidget {
    border: 1px solid #c0c0c0;
    alternate-background-color: #f7f7f7;
}
QHeaderView::section {
    background-color: #f0f0f0;
    padding: 4px;
    border: 1px solid #c0c0c0;
}</string>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
            </property>
            <property name="horizontalScrollMode">
             <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
            </property>
            <property name="showGrid">
             <bool>true</bool>
            </property>
            <property name="gridStyle">
             <enum>Qt::PenStyle::DotLine</enum>
            </property>
            <property name="cornerButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="globalExportCheckBox">
            <property name="text">
             <string>导出全局信息</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="nodesTab">
      <attribute name="title">
       <string>节点信息</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin" stdset="0">
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="nodesGroupBox">
         <property name="title">
          <string>算力节点指标</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin" stdset="0">
           <number>9</number>
          </property>
          <item>
           <widget class="QTableWidget" name="nodesTable">
            <property name="styleSheet">
             <string notr="true">QTableWidget {
    border: 1px solid #c0c0c0;
    alternate-background-color: #f7f7f7;
}
QHeaderView::section {
    background-color: #f0f0f0;
    padding: 4px;
    border: 1px solid #c0c0c0;
}</string>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
            </property>
            <property name="horizontalScrollMode">
             <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
            </property>
            <property name="showGrid">
             <bool>true</bool>
            </property>
            <property name="gridStyle">
             <enum>Qt::PenStyle::DotLine</enum>
            </property>
            <property name="cornerButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="nodesExportCheckBox">
            <property name="text">
             <string>导出节点信息</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="nodesMetricsGroupBox">
            <property name="title">
             <string>选择导出的指标</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="margin" stdset="0">
              <number>9</number>
             </property>
             <property name="spacing">
              <number>8</number>
             </property>
             <item row="0" column="0">
              <widget class="QCheckBox" name="nodeAvgUtilCheckBox">
               <property name="text">
                <string>平均利用率</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="nodeAssignedTasksCheckBox">
               <property name="text">
                <string>已分配任务总数</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="nodeProcessedLoadCheckBox">
               <property name="text">
                <string>已处理总计算量</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="nodeEnergyConsumptionCheckBox">
               <property name="text">
                <string>能耗</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tasksTab">
      <attribute name="title">
       <string>任务信息</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin" stdset="0">
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="tasksGroupBox">
         <property name="title">
          <string>任务指标</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin" stdset="0">
           <number>9</number>
          </property>
          <item>
           <widget class="QTableWidget" name="tasksTable">
            <property name="styleSheet">
             <string notr="true">QTableWidget {
    border: 1px solid #c0c0c0;
    alternate-background-color: #f7f7f7;
}
QHeaderView::section {
    background-color: #f0f0f0;
    padding: 4px;
    border: 1px solid #c0c0c0;
}</string>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
            </property>
            <property name="horizontalScrollMode">
             <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
            </property>
            <property name="showGrid">
             <bool>true</bool>
            </property>
            <property name="gridStyle">
             <enum>Qt::PenStyle::DotLine</enum>
            </property>
            <property name="cornerButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="tasksExportCheckBox">
            <property name="text">
             <string>导出任务信息</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="tasksMetricsGroupBox">
            <property name="title">
             <string>选择导出的指标</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="margin" stdset="0">
              <number>9</number>
             </property>
             <property name="spacing">
              <number>8</number>
             </property>
             <item row="0" column="0">
              <widget class="QCheckBox" name="taskEndToEndDelayCheckBox">
               <property name="text">
                <string>端到端延迟</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="taskComputationTimeCheckBox">
               <property name="text">
                <string>计算时长</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="taskCostCheckBox">
               <property name="text">
                <string>任务成本</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="exportOptionsGroupBox">
     <property name="title">
      <string>导出选项</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin" stdset="0">
       <number>9</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <property name="margin" stdset="0">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>文件类型:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="fileTypeComboBox">
          <item>
           <property name="text">
            <string>CSV文件 (*.csv)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Excel文件 (*.xlsx)</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>输出路径:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QLineEdit" name="outputPathLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="browseButton">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>浏览...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <property name="margin" stdset="0">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>导出</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SimulationExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
